### ISSUE TITLE
most tests fail on Windows7

### ISSUE THREAD

**Issue by mygitplace** [pybee/voc#199]
1. JVM doesn't start:
File "voc\tests\utils.py", line 500, in runAsJava
self.jvm.stdin.flush()
OSError: [Errno 22] Invalid argument
2. File write failed:
File "env\lib\encodings\cp1251.py", line 19, in encode
UnicodeEncodeError: 'charmap' codec can't encode character '\xff' in position 678: character maps to <undefined>

**Comment by cflee** [comment]
@mygitplace is this issue resolved by #200?

**Comment by mygitplace** [comment]
yes, these 2 above were resolved in #200, there still other issues with Windows
